src/build/env.py

```diff
@@ -141,15 +154,23 @@ def __enter__(self):  # type: () -> IsolatedEnvironment
         if 'PATH' in os.environ:
             exe_path.append(os.environ['PATH'])
 
-        self._replace_env('PATH', os.pathsep.join(exe_path))
-        self._replace_env('PYTHONPATH', os.pathsep.join(sys_path))
-        self._replace_env('PYTHONHOME', self.path)
-        self._pop_env('PIP_REQUIRE_VIRTUALENV')
-
         self._symlink_relative(sysconfig.get_path('include'))
         self._symlink_relative(sysconfig.get_path('platinclude'))
         self._symlink_relative(sysconfig.get_config_var('LIBPL'))
 
+        self._symlink_file(sys.executable, os.path.join('bin', 'python'))
```

Comment:
IMHO we should be using here the platform specific folder name and executable, instead of bin/python. We don't want to introduce UNIX-ism on Windows.

Reply:
What if bin/python already exists, can it? Should we overwrite it, I don't think so. Is there any guarantee that ``self.path`` is an empty folder at the start of this run?

Reply:
Shouldn't exist, we are creating a new environment, everything should be empty.

What is the Windows name? AFAIK the executable is also called `python` on Windows.
About the directory name, it is not supposed to be a UNIX-ism. It is merely implementation detail, `IsolcatedEnvironment` is not supposed to be interoperable with other Python environments, that is not the scope of the project, the directory that holds the binaries could be called `environment_binaries` or something else, it is not relevant. What is relevant is that `python` is in `PATH`.

Reply:
If that's the case I prefer a name such as build or something, so it cannot be confused with the Unix bin script folder name.

Reply:
What about `executable`?